SAMtemplates/functions/main.yaml

```diff
@@ -70,6 +67,7 @@ Resources:
         Variables:
           AWS_LAMBDA_EXEC_WRAPPER: /opt/get-secrets-layer
           LOG_LEVEL: !Ref LogLevel
+          GET_STATUS_UPDATES: !Ref ToggleGetStatusUpdates
```

Comment:
should we pass this in at deployment time as an environment variable, or should we retrieve it from ssm at runtime in the lambda?
if we change the ssm param, but not the environment variable does it matter?

Reply:
Calling SSM for the parameter from within the lambda now.

Reply:
Actually, as we discussed, using only the lambda env var now.